api/tsconfig.json

```
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "skipLibCheck": true,
+    "strict": false,
+    "forceConsistentCasingInFileNames": true,
+    "noEmit": true,
+    "esModuleInterop": true,
+    "module": "esnext",
```

```suggestion
    "module": "commonjs",
```
I'm pretty sure you can't use ESM with Node 12.